airmozilla/main/templates/main/home.html

```diff
@@ -1,5 +1,5 @@
 {% extends 'main/main_base.html' %}
-{% set front_page = (events.number == 1) %}
+{% set front_page = (events and events.number == 1) %}
```

Comment:
This is breaking `airmozilla/main/tests/test_views.py:TestPages.test_channels_page` because it now requires that there are events on this page for it to be considered the "front page". 
A channel page, might not have any events (it might [have sub-channels](https://air.mozilla.org/channels/mozfest/)) but it might still be the front page. 

Why did we need this again? The purpose of this "front_page" business is two-fold. To list the live events if there are any and to show the channel title/banner.